aiida/orm/implementation/storage_backend.py

```diff
@@ -63,15 +63,29 @@ def version_profile(cls, profile: 'Profile') -> Optional[str]:
         :raises: `~aiida.common.exceptions.UnreachableStorage` if the storage cannot be accessed
         """
 
+    @classmethod
+    @abc.abstractmethod
+    def initialise(cls, profile: 'Profile', reset: bool = False) -> None:
```

Comment:
The method should return a boolean for if it was created

reset should maybe be a separate method

Reply:
> The method should return a boolean for if it was created

Done